CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ endif()
 set(BUILD_BENCHMARK_LOG "${BUILD_BENCHMARK_LOG}"
   CACHE BOOL "Build binary that benchmarks spdlog speed" FORCE)
 
+if(NOT BUILD_TEST_VALUES)
```

Comment:
Could you rename this variable to `BUILD_GTESTS` as it's really more general than the specific tests we have for values?